### ISSUE TITLE
[Logic]: incorporate rupee rewards for related checks

### ISSUE THREAD

**Issue by foxsouns** [kbranch/Magpie#21]
probably a feature better for autotracker.. but im weird and like tracking the absolute hell out of my stuff, and it bugs me to see "collectable check" on something that i cannot obtain without getting more rupees. could be done with the dev fork, as it sits, by having a rupee reward set a value up by that much, and removing rupees from it when checking a rupee-required check.

(maybe i should stop having ideas: it would take away some of the work you have to put into this lmao)

**Comment by kbranch** [comment]
Yep, definitely a good idea now that the autotracker is set up. It might be a little tricky to perfectly represent how logic works behind the scenes, but just spying on the current rupee count is probably good enough.

**Comment by foxsouns** [comment]
I guess we could have it appear as an OutOfLogic check, with an appended "$" upon it to indicate that you _could_ put this into logic if you can get the money for it; and display your total estimated rupee count either with the check, or on the side. we need to be no fuzzier than 10 rupees (for 980 tool shop, 20/50 rupee reward), but we could probably get away with just that. also need to allow the user to add their own collected rupees, either in the one's digit, or in increments of 10. i feel like focus any more than 10 wouldn't really be helpful, though.
